tests/operators/test_conv2d.py

```diff
@@ -122,15 +125,40 @@ def test_conv2d_channel_last(n, c, h, w, oc, kx, ky, groups, stride, dilations):
         [1, 3, 32, 32, 12, 1, 1],  # kernel 1,
     ],
 )
-@pytest.mark.parametrize("padding", [[0, 0, 0, 0], [1, 2, 1, 2]])
+@pytest.mark.parametrize("padding", [[0, 0], [1, 2]])
 @pytest.mark.parametrize("stride", [[1, 1], [2, 3]])
 @pytest.mark.parametrize("dilations", [[1, 1], [2, 3]])
-def test_conv2d(hidet_op, n, c, h, w, oc, kx, ky, padding, stride, dilations):
+def test_conv2d(n, c, h, w, oc, kx, ky, padding, stride, dilations):
     check_binary(
         a_shape=[n, c, h, w],
         b_shape=[oc, c, kx, ky],
         numpy_op=lambda data, weight: torch_conv2d(data, weight, padding, stride, dilations),
-        hidet_op=lambda data, weight: hidet_op(ops.conv_pad(data, padding), weight, stride=stride, dilations=dilations),
+        hidet_op=lambda data, weight: ops.conv2d(data, weight, padding=padding, stride=stride, dilations=dilations),
+        dtype='float32',
+        atol=2e-5,
+        rtol=2e-5,
+    )
+
+
+@pytest.mark.parametrize(
+    "n, c, h, w, oc, kx, ky",
+    [
+        [1, 3, 32, 32, 12, 3, 3],  # kernel 3,
+        [2, 3, 32, 32, 12, 7, 7],  # kernel 7, batch size 2
+        [1, 3, 32, 32, 12, 1, 1],  # kernel 1,
+    ],
+)
+@pytest.mark.parametrize("padding", [[0, 0], [1, 2]])
+@pytest.mark.parametrize("stride", [[1, 1], [2, 3]])
+@pytest.mark.parametrize("dilations", [[1, 1], [2, 3]])
+def test_conv2d_gemm(n, c, h, w, oc, kx, ky, padding, stride, dilations):
+    check_binary(
```

Comment:
Let's reduce the number of tests here. The previous PR (https://github.com/hidet-org/hidet/pull/283/files) adds 10 more minutes. We can  use the configs like
```
    "n, c, h, w, oc, kx, ky,padding,stride,dilations",
    [
        [1, 3, 32, 32, 12, 3, 3, [0, 0], [1, 1], [1, 1]],  # kernel 3,
        [2, 3, 32, 32, 12, 7, 7, [1, 2], [2, 3], [2, 3]],  # kernel 7, batch size 2
        [1, 3, 32, 32, 12, 1, 1, [0, 0], [2, 3], [1, 1]],  # kernel 1,
    ],
```